wiki/en/Running-a-Server.md

```diff
@@ -83,69 +66,34 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
-
-`sudo systemctl stop jamulus-headless`
-
-#### Running in Registered mode
-
-The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+Many people attribute problems to the Server that are in fact problems with the _Client_. Much depends on the Clients' hard/software setup, the networks that _they_ are on, and whether they are sticking to [Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
 
-~~~
-jamulus --nogui --server \
-        --directoryserver genreServer:port \
-        --serverinfo "yourServerName;yourCity;[country ID]"
-~~~
+**_The capability of the Server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**
 
-**Note**: Semicolon and newline characters are not allowed in `yourServerName` and `yourCity` within the `--serverinfo` argument
+If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
 
-To register with one of the Directories built into the Jamulus Client, replace `genreServer:port` in the example above with one of the following options:
+Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home.
 
+### Bandwidth – do you have enough?
 
-| Genre |   Server Address           |
-|-----------|------------------|
-|**Any Genre 1** |`anygenre1.jamulus.io:22124`|
-|**Any Genre 2** |`anygenre2.jamulus.io:22224`|
-|**Any Genre 3** |`anygenre3.jamulus.io:22624`|
-|**Genre Rock** |`rock.jamulus.io:22424`|
-|**Genre Jazz** |`jazz.jamulus.io:22324`|
-|**Genre Classical/Folk** |`classical.jamulus.io:22524`|
-|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
+Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
 
-You can also specify a [Custom Directory](#3-custom-directory) in the same way from the command line, providing the Server Address in the same format.
 
-#### Running as a Directory
+## Setting up a server
 
-If you wish to run a [Custom Directory](Running-a-Server#3-custom-directory) please see [this guide](Custom-Directories).
+Please see the [Server Installation page](Server-Install).
```

Comment:
When I suggested the headless server install as a separate page, I meant **just** the install and setup steps, so that this would mirror what we do with the client setup pages and the Software Manual. I moved the headless installation stuff into the main manual because I agreed with @pljones that it was too hard to find. So now we're just going around in circles. 

Another problem with what you have done here is that because some of the information in this separate page overlaps with that of the GUI stuff, we will have to resist the temptation to duplicate content or create a mesh of cross-links or includes to avoid the slow death that duplication eventually brings. 

I understand that you think the page is too long, but I think you need to articulate why that's bad.

Reply:
>  I think you need to articulate why that's bad.

If I am a new user who just wants to setup a server, I just want to know how to install a server with a hands one guide (and maybe some little background info) – similar to the client install page. How to do that should be quickly available

Reply:
> How to do that should be quickly available

Yes, and that was @pljones observation too. So we decided to move that information into (and further up) the file, making the assumption that headless servers are the main (but not only) type of server install. 

What you have done means we are not only back to having it in a hard to find separate file, but have also added information into that file that overlaps with  information for GUI users (eg dyndns), forcing them to pogo between pages. 

BTW some of the most popular websites in the world have long scrolling pages (some perhaps >10x longer than the server admin page). Do not worry about length in that sense. There is no evidence whatsoever that people can't scroll. Finding and clicking on links to pages, on the other hand, has plenty of evidence that it can be confusing.

Reply:
Did you look at the (built) version of https://github.com/jamulussoftware/jamuluswebsite/pull/715? I can't tell if you have or not, but it seems like maybe not?

Reply:
BTW you didn't respond to my suggestion about having the headless and GUI install steps (only) on a separate page linked from the right hand nav. It seems to me that if you want a "I just want to install a server and will read/learn about using it later" journey, then that would serve it. That is, mirroring what we do for the client's docs.

Reply:
Ah. Sorry. That is also a valid approach and should have been discussed in #707 ;-).

And no - I haven't looked at the compiled version of this. However the .md looks still huge (especially with the long introduction).

Yes I would like to go the 

> "I just want to install a server and will read/learn about using it later" journey, then that would serve it. 

route.

Reply:
> OK. I suggest you fire up Jekyll locally and look at the changes myself

Yes, I did that a few minutes ago and still think it's a bit too much text before the actual installation starts - I think it still depends on what the user actually wants if he sees this page. If we assume he wants documentation, the current approach is ok. If we assume he actually wants to set up a server, I think we're giving a bit too much theory. 

What about the following compromise for the next release at least:
1. Link the installation paragraph on the nav
2. Try to minimize the text above
Let me open another PR.

Reply:
OK try that on another PR. 

I get what you mean about what the user wants, but do you see my point about our duty to them? What they might _need_ may not be what they think they might _want_. There is a difference.

Reply:
Ok. Closing this PR then in favor of #725